pipelines/push-disk-images-to-marketplaces/README.md

```diff
@@ -0,0 +1,21 @@
+# push-disk-images-to-marketplaces pipeline
+
+Tekton Pipeline to push disk images to various cloud marketplaces
+
+## Parameters
+
+| Name                            | Description                                                                                            | Optional | Default value                                             |
+|---------------------------------|--------------------------------------------------------------------------------------------------------|----------|-----------------------------------------------------------|
+| release                         | The namespaced name (namespace/name) of the Release custom resource initiating this pipeline execution | No       | -                                                         |
+| releasePlan                     | The namespaced name (namespace/name) of the releasePlan                                                | No       | -                                                         |
+| releasePlanAdmission            | The namespaced name (namespace/name) of the releasePlanAdmission                                       | No       | -                                                         |
+| releaseServiceConfig            | The namespaced name (namespace/name) of the releaseServiceConfig                                       | No       | -                                                         |
+| snapshot                        | The namespaced name (namespace/name) of the snapshot                                                   | No       | -                                                         |
+| enterpriseContractPolicy        | JSON representation of the policy to be applied when validating the enterprise contract                | No       | -                                                         |
+| enterpriseContractExtraRuleData | Extra rule data to be merged into the policy specified in params.enterpriseContractPolicy. Use syntax "key1=value1,key2=value2..." 
```

Comment:
15 and 16 should be one line. If you click the three dots top right of this box then view file, you can see it currently does not render properly

Reply:
Thanks!

I've fixed it